lab1.js

```diff
@@ -71,8 +81,31 @@ var sentence2 = 'Come over here so you can scratch my belly.';
  **two** different kinds of loops to implement this. (10 points each.)
  HINT: the 'split' method on String will be useful.
 */
+/*function assert(sentence,split) {
+  	var res=sentence.split("")
+  }
+}*/
+	var sentence1 = 'More food please.';
+var res=sentence1.split(" ");
+sentence1="";
+for(var i=1;i<res.length;i++)
+{
+	sentence1= sentence1+" chirp "
+
+}
+sentence.trim();
+sentence1= sentence1+" chirp. "
```

Comment:
start adding more spaces to your code. So this line would be
sentence1 = sentence1 + " chirp. ";
it makes everything much more readable. more readable = easier to find bugs.